gettsim/parameters/arbeitsl_geld.yaml

```diff
@@ -188,3 +164,41 @@ stundengrenze:
   2012-04-01:
     scalar: 15
     reference: Art. 2 G. v. 20.12.2011 BGBl I S. 2854
+
+anwartschaftszeit:
```

Comment:
I would rename the variable to `anspruchsdauer` and restructure it as follows:

```yaml
    nach_alter:
      0: 12
      50: 15
      55: 18
      ...
    nach_anwartschafszeit:
      12: 6
      18: 8
      ...
      48: 24
    
```

And then use `piecewise_polynomial` as [here](https://github.com/iza-institute-of-labor-economics/gettsim/blob/d1f7b3ff464a74600892d8d42bb7635e3d32c575/gettsim/transfers/wohngeld.py#L128) for both set of parameters and set anspruchsdauer to the minimum of the two values (let me know if it isn't clear enough what I mean).